test/unit/services/fact_parser_test.rb

```diff
@@ -93,6 +92,46 @@ class NetworkFactParserTest < ActiveSupport::TestCase
               'mtu' => 1500,
               'promisc' => false,
               'type' => 'ether'
+            },
+            'ansible_eth1' => {
+              'active' => true,
+              'device' => 'eth1',
+              'macaddress' => '52:54:00:04:55:38',
+              'perm_macaddress' => '52:54:00:04:55:38',
+              'mtu' => 1500,
+              'promisc' => false,
+              'type' => 'ether'
+            },
+            'ansible_eth2' => {
+              'active' => true,
+              'device' => 'eth2',
+              'macaddress' => '52:54:00:04:55:38',
+              'perm_macaddress' => '52:54:00:04:55:39',
+              'mtu' => 1500,
+              'promisc' => false,
+              'type' => 'ether'
+            },
+            'ansible_bond0' => {
+              'active' => true,
+              'device' => 'bond0',
+              'macaddress' => '52:54:00:04:55:38',
+              'ipv4' => {
+                  'address' => '10.10.0.11',
+                  'netmask' => '255.255.0.0',
+                  'network' => '10.10.0.0'
+              },
+              'ipv6' => [
+                  {
+                      'address' => 'fd00::5054:00ff:fe04:5538',
+                      'prefix' => '64',
+                      'scope' => 'host'
+                  }
+              ],
+              'mtu' => 1500,
+              'promisc' => false,
+              'slaves' => %w(eth1 eth2)
+              ],
```

Comment:
the lcosing square bracket here is unexpected

Reply:
Oh, must've been a merge error probably. I'll fix it and run tests on my system before pushing again. Terribly sorry for this!